public/data/subhadiptech.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "Subhadip Das",
+    "bio": "Improving",
+    "avatar": "https://avatars.githubusercontent.com/u/89024718?s=400&u=ff742340a81589cb9dd85fe7c978a68c9e17ae56&v=4",
+    "links": [
+      {
+        "name": "Follow me on GitHub",
+        "url": "http://github.com/Subhadiptech",
```

Comment:
@Subhadiptech Please can you change the `http` to `https`? Thank you.

Reply:
Thanks for the suggestion!